rstb/rstb-op-polarimetric-tools-ui/src/main/java/org/csa/rstb/polarimetric/gpf/ui/CPRVIOpUI.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2015 by Array Systems Computing Inc. http://www.array.ca
```

Comment:
Include your own copyright and license in all files

Reply:
Updated in the latest version.